src/screens/Exchange/Exchange.js

```diff
@@ -660,10 +691,18 @@ class ExchangeScreen extends React.Component<Props, State> {
           onPress={() => this.onOfferPress(offer)}
```

Comment:
shouldn't it check which type is clicked and then trigger `onFiatOfferPress` if it's fiat?

Reply:
927faae